app/frontend/src/components/form/Step2.vue

```diff
@@ -679,7 +679,11 @@ export default {
         window.scrollBy(0, -60); // ugh again
       }
     }
-
+  },
+  mounted() {
+    // Once they've gotten to the form start (step 2) enable the typical "leave site" native browser warning
+    // This gets disabled after form submit in step 6
+    window.onbeforeunload = () => {return true;};
```

Comment:
As this is a one-liner, we can probably just simplify and it'll still do what you think it will do.
```
window.onbeforeunload = () => true;
```

Reply:
Yep, works